main.nf

```diff
@@ -1024,6 +1024,76 @@ process predict_peptides {
      """
 }
 
+/*
+ * STEP 19 - Predict all possible neoepitopes from vcf
+ */
+process predict_possible_neoepitopes {
+    publishDir "${params.outdir}/"
+    echo true
+
+    input:
+     file alleles_file from input_alleles_neoepitope
+     file vcf_file from input_vcf_neoepitope
+
+    output:
+     file "vcf_neoepitopes.csv" into possible_neoepitopes
+    
+    when:
+     params.vcf.length() > 0
+
+    script:
+     """
+     vcf_neoepitope_predictor.py -t ${params.variant_annotation_style} -a ${alleles_file} -minl ${params.peptide_min_length} -maxl ${params.peptide_max_length} -v ${vcf_file} -o vcf_neoepitopes.csv
+     """
+}
+
+/*
+ * STEP 20 - Resolve found neoepitopes
+ */
+process Resolve_found_neoepitopes {
+    publishDir "${params.outdir}/"
+    echo true
+
+    input:
+     file mztab from features_mztab_neoepitopes
+     file neoepitopes from possible_neoepitopes
+
+    output:
+     file "found_neoepitopes.csv" into found_neoepitopes
+    
+    when:
+     params.vcf.length() > 0
```

Comment:
Is there a neater way of testing whether or not a vcf  file was supplied? I'd rather not use if else logic, but a 'when:' statement.

Reply:
For me this step is failing, I would go for when params.include_proteins_from_vcf